<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlItem</class>
 <widget class="QWidget" name="ControlItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>256</width>
    <height>256</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>0</number>
     </property>
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="rhoLabel">
       <property name="text">
        <string>rho</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="rhoDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.300000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="thetaLabel">
       <property name="text">
        <string>theta</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="thetaDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.017453000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="thresholdLabel">
       <property name="text">
        <string>threshold</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="thresholdSpinBox">
       <property name="minimum">
        <number>-9999</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="minLineLengthLabel">
       <property name="text">
        <string>minLineLength</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="minLineLengthDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="maxLineGapLabel">
       <property name="text">
        <string>maxLineGap</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="maxLineGapDoubleSpinBox">
       <property name="decimals">
        <number>10</number>
       </property>
       <property name="minimum">
        <double>-9999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.500000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="select_image_button">
       <property name="text">
        <string>选择图片</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="do_button">
       <property name="text">
        <string>执行</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>rhoDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>10</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thetaDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>thresholdSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minLineLengthDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>163</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maxLineGapDoubleSpinBox</sender>
   <signal>editingFinished()</signal>
   <receiver>do_button</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>280</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
